flavors/python-3.6-minimal-EXASOL-6.2.0/flavor_base/language_deps/packages/apt_get_packages

```diff
@@ -1,4 +1,4 @@
 ca-certificates|20210119~18.04.1
-python3-dev|3.6.7-1~18.04
-python3-distutils|3.6.9-1~18.04
+python3.7-dev|3.6.7-1~18.04
+python3.7-distutils|3.6.9-1~18.04
```

Comment:
I guess this was a mistake?

Reply:
yes, python needs to be removed from that file, but I am surprised that it doesn't complain

Reply:
```suggestion
```